### DISCUSSION TITLE
message: "Invalid file format. Supported formats: ['m4a', 'mp3', 'webm', 'mp4', 'mpga', 'wav', 'mpeg']"

### DISCUSSION THREAD

**Discussion by Ibby-debug** [openai/whisper#1408] [Q&A]
I'm getting this error back while trying to record on a IOS using expo-av; I posted a question on stackoverflow with more details; Would appreciate any help or direction. https://stackoverflow.com/questions/76367660/react-native-using-expo-av-ios-mp4-file-openais-audio-transcriptions-invalid-fi

**Comment by ryanheise** [comment]
That error message is not a result of whisper, as whisper supports a much larger set of audio formats recognised by ffmpeg. I think you might be asking about Open AI's online API which wraps an endpoint around whisper but it provides a more limited set of features than whisper itself. In any case they do have their own discussion forum over there if you have a question about their service.

**Comment by benjieperez** [comment]
I also experience this using different mp3 format but returns invalid file format error.

This was a chunk of audios in 2 mins segments, sometimes its working sometimes not.

Done Transcribing temp/648fd37645c1a2374c099eb5_1.mp3
Done Transcribing temp/648fd37645c1a2374c099eb5_120001.mp3
Done Transcribing temp/648fd37645c1a2374c099eb5_240001.mp3
Retrying Transcription: temp/648fd37645c1a2374c099eb5_360001.mp3
{'error': {'message': "Invalid file format. Supported formats: ['m4a', 'mp3', 'webm', 'mp4', 'mpga', 'wav', 'mpeg']", 'type': 'invalid_request_error', 'param': None, 'code': None}, 'process_time': 2.6553714275360107}

**Comment by serhii-bilyk** [comment]
I have same issue on iPhone.

**Comment by ilyak1990** [comment]
For me specifically it was on iPhone, I was saving a valid .wav file (was working when I tested it) then I used a file type detector tool to find out it was actually some other file format that apple was saving it to, you can either convert to and from using node library **ffmpeg** or for ihpone specifically save it as a .m4a file instead of .wav

**Comment by Nolanjue** [comment]
if anyone else is  still going through this issue, i think it might be data corruption from passing the audio file to the server. This discussion had a solution: https://stackoverflow.com/questions/77602701/openai-whisper-api-invalid-file-format-on-aws-lambda
